src/index.ts

```diff
@@ -132,26 +132,22 @@ const parseFontFilename = (
   { srcUrlToFilename = filename => filename }: Options,
   result: Result,
 ): string | null => {
-  const urlNode = parser
-    .parse(srcDecl.value)
-    .nodes.find(
-      (node): node is Func => node.type === `func` && node.name === `url`,
-    )
-  if (!urlNode) {
+  const url = urlValue(srcDecl.value)
+  if (!url) {
     srcDecl.warn(result, `No url`)
     return null
   }
 
-  return srcUrlToFilename(urlNode.nodes.map(stringifyWithoutQuotes).join(``))
+  return srcUrlToFilename(url)
 }
 
 const parseFontFamily = (
   familyDecl: Declaration,
   { fontTypes }: Options,
   result: Result,
 ): Pick<FontFaceValues, `family` | `type`> | null => {
-  const family = stringifyWithoutQuotes(parser.parse(familyDecl.value))
-  if (isFallbackFontFamily(family)) {
+  const family = fontFamilyNameValue(familyDecl.value)
+  if (!family || isFallbackFontFamily(family)) {
```

Comment:
Can we separate out the `!family` case and do `familyDecl.warn('Bad font-family')`? Reason being that this is an unexpected case while the latter part of the condition is expected to happen

Reply:
Done, I've also grouped the test cases relevant to this code path.